src/main/java/net/minecraftforge/fmp/capabilities/PartAttachCapabilitiesEvent.java

```diff
@@ -0,0 +1,22 @@
+package net.minecraftforge.fmp.capabilities;
+
+import net.minecraftforge.event.AttachCapabilitiesEvent;
+import net.minecraftforge.fmp.multipart.IMultipart;
+
+public class PartAttachCapabilitiesEvent extends AttachCapabilitiesEvent
+{
```

Comment:
If this is needed it'd be moved into the subclass of the event just like everything else not it's own class.